### DISCUSSION TITLE
scrapli.exceptions.ScrapliTimeout: timed out sending interactive input to device

### DISCUSSION THREAD

**Discussion by gaffar10** [carlmontanari/scrapli#172] [General]
@carlmontanari

Dears, 

Can someone please help to provide me the alternate option to use Scrapli without having the privilege level 15 password?.

 I am using scrapli on the Nornir framework and the script works fine if I configure the privilege level 15 password on the Cisco Device.  However, on the production devices, by default it goes to the User EXEC(Switch>) mode, then I have to manually type "enable" to go into the Privileged EXEC mode(Switch#).

Is there any alternate option, to pass the enable password similar to the get.pass method? 

If auth_secondary is the only option, may I know where to define the Enable password? is it possible to encrypt rather than specifying in clear text. 

Thanks in Advance

**Comment by carlmontanari** [comment]
This is exactly what `auth_secondary` is for -- if you set your password there scrapli will auto acquire "privilege exec" (in cisco parlance -- in other platforms whatever is the primary privilege level for running operational commands). 

You can pass scrapli that password however you want, thats a problem outside of scrapli really -- you can use get pass, environment variables, ansible/hashi/whatever vault, or whatever you want.

**Reply by gaffar10** [reply]
Thank you very much for your swift response Carl.
